ksql-engine/src/test/java/io/confluent/ksql/util/KafkaTopicClientTest.java

```diff
@@ -124,6 +129,58 @@ public void shouldNotFailIfTopicAlreadyExistsWhenCreating() throws InterruptedEx
     KafkaTopicClient kafkaTopicClient = new KafkaTopicClientImpl(adminClient);
     kafkaTopicClient.createTopic(topicName1, 1, (short)1, false);
     verify(adminClient);
+
+  }
+
+
+  // Compares two collections of 'kafka.clients.admin.NewTopic' objects to verify that they are
+  // the same. The NewTopic.equals method isn't implemented so it just does reference equality,
+  // which is not what we want in these tests.
+  class NewTopicComparator implements Comparator<Collection<NewTopic>> {
```

Comment:
`private static class`, and could we move this to the bottom of the file please?